image-builder-cli.spec

```diff
@@ -3,8 +3,12 @@
 %bcond_with tests
 %bcond_with relax_requires
 
-# The minimum required osbuild version
-%global min_osbuild_version 129
+# The minimum required osbuild version, note that this used to be 129
+# but got bumped to 138 for librepo support which is not strictly
+# required. So if this needs backport to places where there is no
+# recent osbuild available we could simply make --use-librepo false
+# and go back to 129.
```

Comment:
Is this a realistic issue?  I guess it might be if you're `go install`ing ib-cli, but in that case, you're on your own and the spec file doesn't matter.

I mean, there's no way this spec file will be used to **officially** build ib-cli in a distro where a new enough osbuild wont be, so I think this note isn't relevant.

I might be missing something though.